### PR TITLE
Linguist blog update

### PR DESCRIPTION
**Pull request by SrivatsanSenthilkumar**
I caught some errors that I did not catch earlier and fixed them. Sorry I did not catch them earlier.

### CODE REVIEW

File: posts/2023/july/linguist/2023-07-13-linguist.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ With the dictionary feature, you can bookmark translations and find them later,
...
 
-This features makes Linguist ideal tool for traveling, because it saves your internet bandwidth and enables translation even if you are offline due to lack of a data connection. You can even find words using the search functionality of the offline dictionary.
+These features makes Linguist the ideal tool for traveling, because it saves your internet bandwidth and enables translation even if you are offline due to the lack of a data connection. You can even find words using the search functionality of the offline dictionary.
 
-Linguist works on smartphones, but it does not work on iOS because iOS does not have a browser that supports the WebExtensions API, only Safari. I'm sad to see that my friends who use an iPhone can't even translate the restaurant menu and they look surprised when I translate sites on my Android with Linguist.
+Linguist works on smartphones, but it does not work on iOS because iOS does not have a browser that supports the WebExtensions API, only Safari. I'm sad to see that my friends who use an iPhone can't even translate the restaurant menu and they look surprised when I translate sites on my Android device with Linguist.
```
Comment (vitonsky) [inline posts/2023/july/linguist/2023-07-13-linguist.md:60]: Let's revert this change

File: posts/2023/july/linguist/2023-07-13-linguist.md [hunk 2]
```diff
@@ -68,6 +68,6 @@ Try Linguist today and share it with your friends, followers and community.
 - [Firefox](https://addons.mozilla.org/addon/linguist-translator/)
   - Read the guide [how to install on android](https://github.com/translate-tools/linguist#android)
 
-Rate Linguist on your browser's store. Let's make Linguist more popular together. Create activity on social media, share links to Linguist to help promote a good tool you like.
+Rate Linguist on your browser's store. Let's make Linguist more popular together. Create activity on social media, share links to Linguist to help promote a good tool that you use and like.
```
Comment (vitonsky) [inline posts/2023/july/linguist/2023-07-13-linguist.md:71]: Too hard to read for me. Let's leave as is

**Comment by SrivatsanSenthilkumar** [comment]
Done reverting

**Comment by SrivatsanSenthilkumar** [comment]
Hopefully the people who were complaining over the grammar earlier will now start recommending linguist too.

**Comment by vitonsky** [comment]
> Hopefully the people who were complaining over the grammar earlier will now start recommending linguist too.

Yeah, i hope too. I appreciate you have decided to fix the typos. The people in community who care and who ready to contribute motivates me!
